content/applications/marketing/events/create_events.rst

```diff
@@ -0,0 +1,427 @@
+If the checkbox is *not* ticked for this setting, Odoo will present the question for every attendee
+that is connected to that reservation/registration.
+
+Next, select a :guilabel:`Question Type` option. The following are the :guilabel:`Question Type`
+options:
+
+- :guilabel:`Selection`: provide answer options to the question for registrants to choose from.
+  Selectable answer options are added in the :guilabel:`Answers` column at the bottom of the pop-up
+  window. This is the only :guilabel:`Question Type` where the :guilabel:`Answers` tab appears.
+  There are no correct or incorrect answers.
+- :guilabel:`Text Input`: lets the users enter a custom response to the question in a text field.
+- :guilabel:`Name`: provides registrants with a field for them to enter their name.
+- :guilabel:`Email`: provides registrants with a field for them to enter their email address.
+- :guilabel:`Phone`: provides registrants with a field for them to enter their phone number.
+- :guilabel:`Company`: provides registrants with a field for them to enter a company they are
+  associated with.
+
+Once all the desired configurations have been entered, either click :guilabel:`Save & Close` to save
+the question, and return to the :guilabel:`Questions` tab on the event form, or click
+:guilabel:`Save & New` to save the question and immediately create a new question on a new
+:guilabel:`Create Question` pop-up window.
+
+As questions are added to the :guilabel:`Questions` tab, the informative columns showcase the
+configurations of each question.
+
+The informative columns are the following:
+
+- :guilabel:`Title`
+- :guilabel:`Mandatory`
+- :guilabel:`Once per Order`
+- :guilabel:`Type`
+- :guilabel:`Answers` (if applicable)
+
+For :guilabel:`Selection` and :guilabel:`Text Input` types, a clickable :guilabel:`📊 Stats` icon
+appears on the right side of the question line. When clicked, Odoo reveals a separate page,
+showcasing the response metrics to that specific question.
+
+To delete any question from the :guilabel:`Questions` tab, simply click the :guilabel:`🗑️ (trash
+can)` icon on the corresponding question line.
+
+There is no limit to how many questions can be added in the :guilabel:`Questions` tab of an event
+form.
+
+.. _create_events/event_form/notes:
+
+Notes tab
+---------
+
+In the :guilabel:`Notes` tab of an event form, users can leave detailed internal notes and/or
+event-related instructions/information for attendees.
+
+.. image:: create_events/notes-tab.png
+   :align: center
+   :alt: Typical notes tab on an event form in the Odoo Events application.
+
+In the :guilabel:`Note` field of the :guilabel:`Notes` tab, users can leave internal notes for other
+event employees, like "to-do" lists, contact information, instructions, and so on.
+
+In the :guilabel:`Ticket Instructions` field of the :guilabel:`Notes` tab, users can leave specific
+instructions for people attending the event, like how to enter, when the venue closes, and so on.
```

Comment:
Explaining where these Ticket Instructions display is more important to detail than extraneous use-cases 😉
```suggestion
instructions for people attending the event that appear on the attendees ticket.
```